amd_openvx/openvx/ago/ago_kernel_api.cpp

```diff
@@ -1867,10 +1867,17 @@ int ovxKernel_HarrisCorners(AgoNode * node, AgoKernelCommand cmd)
     // INFO: use VX_KERNEL_AMD_HARRIS_SOBEL_* kernels to compute Gx^2, Gx*Gy, Gy^2
     //       use VX_KERNEL_AMD_HARRIS_SCORE_* kernels to compute Vc
     //       use VX_KERNEL_AMD_HARRIS_MERGE_SORT_AND_PICK_XY_HVC kernel for final step
-    //       disable buffer merging for HarrisCorners
-    if(node->attr_affinity.device_type == AGO_KERNEL_FLAG_DEVICE_GPU){
-        agoSetEnvironmentVariable("AGO_BUFFER_MERGE_FLAGS", "1");
+
+// TBD -- disable buffer merging for HarrisCorners as buffer merging is not working for this node in OpenCL Flow
+#if ENABLE_OPENCL
```

Comment:
add || ENABLE_HIP to #if as this workaround is needed for HIP as well

Reply:
I saw the HIP backend failure on jenkins.